JUnit5 method parameter injection fails if "TestEnricher.resolve" returns null

This test in "arquillian-extension-warp" fails after updating to 1.9.2 or newer: https://github.com/arquillian/arquillian-extension-warp/blob/main/ftest/src/test/java/org/jboss/arquillian/warp/ftest/BasicWarpTest.java

The problem is introduced by #608

The error is:
```
[ERROR] org.jboss.arquillian.warp.ftest.BasicWarpTest.test -- Time elapsed: 0.140 s <<< ERROR!
java.lang.NullPointerException
        at org.jboss.arquillian.junit5.MethodParameterObserver.injectParameters(MethodParameterObserver.java:68)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
        at org.jboss.arquillian.core.impl.EventContextImpl.invokeObservers(EventContextImpl.java:103)
        at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:90)
        at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createContext(ContainerEventController.java:128)
        at org.jboss.arquillian.container.test.impl.client.ContainerEventController.createBeforeContext(ContainerEventController.java:114)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
        at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
        at org.jboss.arquillian.test.impl.TestContextHandler.createTestContext(TestContextHandler.java:116)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
        at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
        at org.jboss.arquillian.test.impl.TestContextHandler.createClassContext(TestContextHandler.java:83)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
        at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
        at org.jboss.arquillian.test.impl.TestContextHandler.createSuiteContext(TestContextHandler.java:69)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.jboss.arquillian.core.impl.ObserverImpl.invoke(ObserverImpl.java:86)
        at org.jboss.arquillian.core.impl.EventContextImpl.proceed(EventContextImpl.java:95)
        at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:134)
        at org.jboss.arquillian.core.impl.ManagerImpl.fire(ManagerImpl.java:106)
        at org.jboss.arquillian.test.impl.EventTestRunnerAdaptor.before(EventTestRunnerAdaptor.java:115)
        at org.jboss.arquillian.junit5.ArquillianExtension.beforeEach(ArquillianExtension.java:63)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
```

It happen here: https://github.com/arquillian/arquillian-core/blob/c2ad7bf56beb52243cdb50c30d451514d9cee55f/junit5/core/src/main/java/org/jboss/arquillian/junit5/MethodParameterObserver.java#L68


It is caused by a `org.jboss.arquillian.test.spi.TestEnricher `implementation in the warp test suite that returns "null" in `resolve`: https://github.com/arquillian/arquillian-extension-warp/blob/8751718bee1d257b897aae85bb814ae2416ccc7b/ftest/src/test/java/org/jboss/arquillian/warp/ftest/commandBus/TestCaseEnricher.java#L47

This is easily fixable in warp by returning an empty array instead, but I want to bring it to your attention: a simple null check should be sufficient. I could send a pull request if you think so.

--- a/junit5/core/src/main/java/org/jboss/arquillian/junit5/MethodParameterObserver.java
+++ b/junit5/core/src/main/java/org/jboss/arquillian/junit5/MethodParameterObserver.java
@@ -76,9 +76,11 @@ public class MethodParameterObserver {
             final Collection<TestEnricher> testEnrichers = serviceLoader.get().all(TestEnricher.class);
             for (TestEnricher enricher : testEnrichers) {
                 final Object[] values = enricher.resolve(testMethod);
-                for (int i = 0; i < values.length; i++) {
-                    if (values[i] != null) {
-                        methodParameters.add(i, values[i]);
+                if (values != null) {
+                    for (int i = 0; i < values.length; i++) {
+                        if (values[i] != null) {
+                            methodParameters.add(i, values[i]);
+                        }
                     }
                 }
             }

